x-pack/plugins/apm/server/lib/alerts/chart_preview/get_transaction_error_rate.ts

```diff
@@ -34,13 +33,13 @@ export async function getTransactionErrorRateChartPreview({
   const query = {
     bool: {
       filter: [
-        { range: rangeFilter(start, end) },
         { term: { [PROCESSOR_EVENT]: ProcessorEvent.transaction } },
         ...(serviceName ? [{ term: { [SERVICE_NAME]: serviceName } }] : []),
         ...(transactionType
           ? [{ term: { [TRANSACTION_TYPE]: transactionType } }]
           : []),
```

Comment:
Tangent: Would helpers like these be helpful or not?

```suggestion
        ...serviceNameQuery(serviceName),
        ...transactionTypeQuery(transactionType)
```

Reply:
Probably, though something like a more generic `termQuery(field, value)` might be more widely applicable.